query/language.peg

```diff
@@ -192,7 +192,7 @@ METRIC_NAME <- IDENTIFIER
 TAG_NAME <-    IDENTIFIER
 # TODO - may be refactored later.
 TIMESTAMP <-   NUMBER_NATURAL / STRING
-IDENTIFIER <-  "`" CHAR* "`" / !KEYWORD ID_SEGMENT ("." ID_SEGMENT)*
+IDENTIFIER <-  "`" CHAR* "`" / !(KEYWORD !ID_SEGMENT) ID_SEGMENT ("." ID_SEGMENT)*
```

Comment:
Won't this mean that an IDENTIFIER will match something that goes `KEYWORD ID_SEGMENT`?

Since `KEYWORD` would consume `KEYWORD`, and them `ID_SEGMENT` would match `ID_SEGMENT`, the match `(KEYWORD !ID_SEGMENT)` would fail so `!(KEYWORD !ID_SEGMENT)` would succeed, and then you'd match `ID_SEGMENT` on the `KEYWORD` at the beginning.